libibverbs/libibverbs.map.in

```diff
@@ -215,7 +215,6 @@ IBVERBS_PRIVATE_@IBVERBS_PABI_VERSION@ {
 		ibv_cmd_req_notify_cq;
 		ibv_cmd_rereg_mr;
 		ibv_cmd_resize_cq;
-		ibv_get_fw_ver;
```

Comment:
Don't you need to change IBVERBS_PABI_VERSION too?

Reply:
Yes.. And we really should have a CI test for this, I just don't know how to write it.

Reply:
ABI dumper like we have in stable@?

Reply:
I have bumped the private version to be 33 as of current library one, a CI can be really helpful here.